frontend/user-guide/MARKETPLACE_INTEGRATION.md

```diff
@@ -2,80 +2,9 @@
 
 # Marketplace Integration
 * [Becoming a Marketplace vendor](#becoming_a_marketplace_vendor)
-* [Pre-requistites](#prerequisites)
-* [Creating your first solution](#creating_your_first_solution)
-* [Marketplace Concepts](#marketplace_concepts)
-    * [Plans & Features](#plans_and_features)
-        * [Service Features](#service_features)
-        * [Service Plans](#service_plans)
 
 ## <a name="becoming_a_marketplace_vendor">Becoming a Marketplace vendor</a>
-In order to integrate Datashare with Google Cloud Marketplace, you must first sign up to become a [Google Cloud Marketplace vendor](https://cloud.google.com/marketplace/docs/partners). Once you have been approved as a partner you may then list your data solutions within Marketplace.
+In order to integrate Datashare with Google Cloud Marketplace, you must first sign up to become a [Google Cloud Marketplace vendor](https://cloud.google.com/marketplace/docs/partners). Once you have been approved as a partner you may then list your data solutions within Marketplace. The on-boarding team will provide you with access to the Producer Portal or the Partner Portal which you will use to create your marketplace solutions. Depending on which they provide you access to follow the link to the respective instructions below:
```

Comment:
Might be good to provide a quick description or link to the differences b/n Partner and Producer portals.